pkg/destroy/powervs/powervs.go

```diff
@@ -520,34 +524,32 @@ func (o *ClusterUninstaller) loadSDKServices() error {
 		return fmt.Errorf("loadSDKServices: loadSDKServices: o.piSession is nil")
 	}
 
-	ctx, _ := o.contextWithTimeout()
-
-	o.instanceClient = instance.NewIBMPIInstanceClient(ctx, o.piSession, o.ServiceGUID)
+	o.instanceClient = instance.NewIBMPIInstanceClient(context.Background(), o.piSession, o.ServiceGUID)
```

Comment:
I see this client is weirdly designed. So the context you pass here when creating the client will be used for all the subsequent client calls?

Reply:
Yep!

Reply:
Nice job hunting that one down.

Reply:
Thanks!  I was worried I would never find the issue...